cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -194,41 +140,14 @@ public CgmesExportContext(Network network, ReferenceDataProvider referenceDataPr
             topologyKind = networkTopologyKind(network);
         }
         scenarioTime = network.getCaseDate();
-        CgmesMetadataModels models = network.getExtension(CgmesMetadataModels.class);
-        if (models != null) {
-            models.getModelForSubset(CgmesSubset.EQUIPMENT).ifPresent(eq -> prepareExportedModelFrom(exportedEQModel, eq));
-            models.getModelForSubset(CgmesSubset.STEADY_STATE_HYPOTHESIS).ifPresent(ssh -> prepareExportedModelFrom(exportedSSHModel, ssh));
-            models.getModelForSubset(CgmesSubset.TOPOLOGY).ifPresent(tp -> prepareExportedModelFrom(exportedTPModel, tp));
-            models.getModelForSubset(CgmesSubset.STATE_VARIABLES).ifPresent(sv -> prepareExportedModelFrom(exportedSVModel, sv));
-        }
         addIidmMappings(network);
-    }
 
-    /**
-     * Set for each exported model the profile relative to the cim version.
-     * @param cim The cim version from which depends the models profiles uri.
-     */
-    private void initializeExportedModelProfiles(CgmesNamespace.Cim cim) {
-        if (cim.hasProfiles()) {
-            exportedEQModel.setProfile(cim.getProfileUri("EQ"));
-            exportedTPModel.setProfile(cim.getProfileUri("TP"));
-            exportedSVModel.setProfile(cim.getProfileUri("SV"));
-            exportedSSHModel.setProfile(cim.getProfileUri("SSH"));
+        if (network.hasProperty(BOUNDARY_EQ_ID_PROPERTY)) {
+            setBoundaryEqId(network.getProperty(BOUNDARY_EQ_ID_PROPERTY));
+        }
+        if (network.hasProperty(BOUNDARY_TP_ID_PROPERTY)) {
+            setBoundaryTpId(network.getProperty(BOUNDARY_TP_ID_PROPERTY));
         }
-    }
-
-    /**
-     * Update the model used for the export according to the network model.
-     * All the metadata information will be duplicated, except for the version that will be incremented.
-     * @param exportedModel The {@link CgmesMetadataModel} used for the export.
-     * @param fromModel The {@link CgmesMetadataModel} attached to the network.
-     */
-    private void prepareExportedModelFrom(CgmesMetadataModel exportedModel, CgmesMetadataModel fromModel) {
-        exportedModel.setDescription(fromModel.getDescription());
-        exportedModel.setVersion(fromModel.getVersion() + 1);
```

Comment:
Why the exported model does no longer increment the version ?

Reply:
The metadata models in the Network are never updated after export (this is by design). To obtain the version of the exported files one must read the contents of the output.

I've prepared a unit test that I think demonstrates and clarifies how this CGM export can be used. It is inside `CommonGridModelExportTest::testFaraoUseCase`.